src/library_glfw.js

```diff
@@ -616,44 +619,46 @@ var LibraryGLFW = {
       event.preventDefault();
     },
 
-    onCanvasResize: (width, height) => {
+    onCanvasResize: (width, height, framebufferWidth, framebufferHeight) => {
```

Comment:
Would be good to document here why there are two pairs of widthxheight and how they differ.

Reply:
I just did.